wleklinskimateusz/tutoring-manager
# chore: setup next app

closes #1 

## Checklist

- [x] all the tests/checks passed

## Changes made
- added TS config 
- ESLint new flat config (with a few plugins), 
- Prettier,
- PR template file,
- `.vscode` shared settings with plugins recommendation
- opt in react-compiler plugin
- defaulted to force-cache on fetch function

## How to test (if applicable)
- clone the repo check out if everything is working correctly
- check if you would prefer to setup up something differently
